addons/calendar/static/src/js/services/calendar_notification_service.js

```diff
@@ -61,11 +61,12 @@ export const calendarNotificationService = {
                             {
                                 name: env._t("Details"),
                                 onClick: async () => {
-                                    await action.doAction(
-                                        "calendar.action_calendar_event_notify",
-                                        {
-                                            resId: notif.event_id,
-                                        }
+                                    await action.doAction({
+                                        type: 'ir.actions.act_window',
+                                        res_model: 'calendar.event',
+                                        res_id: notif.event_id,
+                                        views: [[false, 'form']],
+                                    }
```

Comment:
Why can't you update the parameters to keep using action_calendar_event_notify but fix the call?
action_calendar_event_notify has no domain or context, active_id could be passed an the domain could use that. 
action_calendar_event_notify is not super helpful so we could drop it in master and use the same strategy as you.

Reply:
No matter what I tried, I couldn't open the event with `calendar.action_calendar_event_notify` and as you said it's only used once elsewhere to create a new event so no more info there + I didn't want to modify too much in Stable but I could had a todo/ remove it in master during the FW port if you believe it's best.

Reply:
Yes, that would be nice. We don't need it anymore.